grammars/weave_md.cson

```diff
@@ -48,6 +48,31 @@
                 ]
       'end' : '$'
     }
+    {
+    'begin': '^([`~]{3,})(@(docs|autodocs|ref|meta|index|content|example|repl|eval|setup|raw))\\s+(\\d+)?\\s*$'
```

Comment:
It would probably be better to merge this with the julia `'^([`~]{3,})(\\{|\\{\\.|)(julia)(;|)\\s*(.*?)(\\}|)\\s*$'` regex above instead adding another almost identical one.

Reply:
I thought about that, but that regex gets pretty complicated... You're probably right though.